app/messages/new_announcement.sms.erb

```diff
@@ -1,5 +1 @@
-<%= t :subject, "New announcement: %{title}, %{course}", :title => asset.title, :course => asset.context.name %>
-
-<%= HtmlTextHelper.strip_and_truncate(asset.message, :max_length => 50) %>
-
-<%= t :details, "More info at %{url}", :url => HostUrl.context_host(asset.context) %>
\ No newline at end of file
+(BZ) <%- asset.title %> <%= HostUrl.context_host(asset.context) %>
```

Comment:
I think we should try: ```(BZ) <% asset.title %>: <%= HtmlTextHelper.strip_and_truncate(asset.message, :max_length => 100) %>  <%= HostUrl.context_host(asset.context) %>```  Ideally, we would calculate the amount of the actual message to show as an SMS length minus the asset title + URL length + (BZ) length.

I don't know what this URL looks like, but it seems like we don't need to go through bit.ly?

Last, is there a typo with this fragment (an extra dash at the beginning)? ```<%- asset.title %>```